crates/core/src/ring/location.rs

```diff
@@ -26,17 +26,19 @@ impl Location {
         match addr.ip() {
             std::net::IpAddr::V4(ipv4) => {
                 let octets = ipv4.octets();
-                let combined_octets = (u32::from(octets[0]) << 16)
-                    | (u32::from(octets[1]) << 8)
-                    | u32::from(octets[2]);
-                Location(combined_octets as f64 / 16777215.0) // 2^24 - 1
+                let combined_octets = (u64::from(octets[0]) << 16)
```

Comment:
is necessary to convert this to u64 and shifft the bits anymore? can't we directly hash the different octets?

Reply:
hmm, yes - I think you're right, will make the change.

Reply:
@iduartgomez I made this simplification.